dev/tools/localization/gen_l10n.dart

```diff
@@ -131,7 +131,7 @@ String generatePluralMethod(Message message, AppResourceBundle bundle) {
           argValue = argValue.replaceAll('#${placeholder.name}#', '\${${placeholder.name}}');
         }
       }
-      pluralLogicArgs.add("      ${pluralIds[pluralKey]}: '$argValue'");
+      pluralLogicArgs.add('      ${pluralIds[pluralKey]}: "$argValue"');
```

Comment:
Is this fixing a different issue?

Reply:
Yes, it's allowing us to have apostrophes in plural strings. It was not working in the latest version. 

Let me add it to the description of the PR.

Reply:
Are we missing some tests?

Reply:
https://github.com/flutter/flutter/pull/54185 Added more test coverage here.